module-4/backend/week_one.md

```diff
@@ -5,16 +5,56 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+    How you can call functions in a class, how to organize code better and more efficiently, and that you can call a function       
+    and set it to a variable and then just call that variable like it was the function with parethesis. 
+    
 2. What are some tools to help debug JavaScript code?
+    pryjs
+    debugger
+
 3. What are some tools you need in order to unit test your JavaScript?
-4. What is the syntax for invoking a function?
+    chai (more specifically assert)
+    the file your aquiring 
+    mocha
+    it helps if npm test is set up 
+    
+
+ 4. What is the syntax for invoking a function?
+    function()
+
 5. What's the difference between `==` and `===` in JavaScript?
+    == is equal
+    == is equal in kind and type 
+    
 6. What's the difference between asynchronous and synchronous JavaScript? 
+    syncronous runs things all at once and there for does one thing, then the next 
+    asyncronous runs things as to be time efficient. if there are three things and one is an api call and the others are  
+    console.logs. when it gets to the api call it'll start the call then go on to make the console.log then when the info 
+    returns will display the api call information.. 
+
 7. How do we setup a route when creating an API with Node and Express?
+    app -> being the server file you required 
+    app.get('/', (request, resposne) => {
+      FoodsController(request, response)
+    })
 8. What do we use Knex for?
```

Comment:
Be able to expand on this more. Just for making databases? How does it relate to any other tools you know? Is it good for anything else?